### ISSUE TITLE
Add prop "onActivePageClick()"

### ISSUE THREAD

**Issue by DmitriyWebDev** [AdeleD/react-paginate#340]
In the case when I need handle click on an active item in the pagination, I cannot do that using pagination props.

I suggest to add optional prop - "onActivePageClick()", which will be called on active item click.

**Comment by DmitriyWebDev** [comment]
My pull request - https://github.com/AdeleD/react-paginate/pull/341

**Comment by DmitriyWebDev** [comment]
@AdeleD , can you please take a look at this PR, I am really waiting for you to reply/merge it.

**Comment by AdeleD** [comment]
Hello,
It has been implemented in version 1.7.1 with a more generic name.
So now, you can do something like:
```javascript
<ReactPaginate onPageActive={(activeItem) => {
    // Do something with the activeItem
}} />
```
